tests/integration/pause.tcl

```diff
@@ -0,0 +1,44 @@
+start_server {tags {"pause external:skip"}} {
```

Comment:
I would add this to the failover integration tests.

Reply:
@madolson Do I need to improve the relevant test contents in this PR?